mmaction/models/backbones/resnet3d_slowfast.py

```diff
@@ -373,7 +373,9 @@ class ResNet3dSlowFast(nn.Module):
         resample_rate (int): A large temporal stride ``resample_rate``
             on input frames, corresponding to the :math:`\\tau` in the paper.
             i.e., it processes only one out of ``resample_rate`` frames.
-            Default: 16.
+            We resample frames beforehand in data pipeline with a interval
+            of 2, therefore ``resample_rate=8`` is equavalent to
```

Comment:
If the interval is configurable, then stating the correct way of computing the effective resample rate is good enough.

Claiming that interval = 2 is not supposed to be here.